### PR TITLE
[MIG][16.0] server_environment_delivery

### PR DESCRIPTION
**Pull request by mmequignon**

### CODE REVIEW

**Comment by ajaniszewska-dev** [comment]
we miss "migration to 16.0" commit - imo we should rename "pre-commit stuff" commit into that.

**Review by simahawk — COMMENTED** [review]
LG
@mmequignon pls update commits

File: server_environment_delivery/models/delivery_carrier.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _name = "delivery.carrier"
+    _inherit = ["delivery.carrier", "server.env.techname.mixin", "server.env.mixin"]
+
+    tech_name = fields.Char(required=False)
```
Comment (simahawk) [inline server_environment_delivery/models/delivery_carrier.py:11]: you can drop this as the field is not required anymore
https://github.com/OCA/server-env/blob/16.0/server_environment/models/server_env_tech_name_mixin.py

**Review by simahawk — APPROVED** [review]

**Comment by rousseldenis** [comment]
@mmequignon Thanks for this. For the next time could you separate pre-commit changes from migration ones in separate commit to avoid these changes in migration commit: 
image

**Comment by mmequignon** [comment]
> @mmequignon Thanks for this. For the next time could you separate pre-commit changes from migration ones in separate commit to avoid these changes in migration commit: image

hello @rousseldenis 
Yeah well, I can split it.
I actually addressed this comment https://github.com/OCA/delivery-carrier/pull/626#issuecomment-1491564681
:blush:

**Comment by simahawk** [comment]
> @mmequignon Thanks for this. For the next time could you separate pre-commit changes from migration ones in separate commit to avoid these changes in migration commit: image

I don't consider this as a pre-commit change. This is just pre-commit doing something for you when you migrate.
Nowadays we should always have that folder generated on migration otherwise other PRs depending on this might be broken :wink: 


/ocabot merge nobump

**Comment by simahawk** [comment]
@mmequignon can you rebase and check the build pls?

**Review by florian-dacosta — APPROVED** [review]
@mmequignon kind reminder, if you could do a rebase so we can merge this, it would be nice !

**Comment by mmequignon** [comment]
> @mmequignon kind reminder, if you could do a rebase so we can merge this, it would be nice !

Thanks for the reminder, it is done :pray:

**Comment by florian-dacosta** [comment]
@simahawk can you retry to merge please ?

**Comment by bguillot** [comment]
thank you !

**Pull request merged**